tests/test_hovernet_loss.py

```diff
@@ -126,32 +126,32 @@ def test_shape_generator(num_classes=1, num_objects=3, batch_size=1, height=5, w
 
```

Comment:
Hi @JHancox, that's a great point but we are essentially using different kernel here. I can normalize the kernels to be in the same order at least but I don't know how much it will help.

Reply:
@JHancox I have updated the kernels to give us the proper gradients. Now they are in the same order of previous implementation but essentially different. I think if the hovernet algorithm is a robust enough, a slight difference in the definition of gradients in one of the six losses shouldn't affect the overall performance. Don't you think so?

Reply:
It's probably okay. Out of interest, what is the reason for changing the kernel?

Reply:
To generalize the sobel gradient to 3D images and also use separable kernels which are faster than the original implementation.